cmd/argocd/commands/app.go

```diff
@@ -732,14 +732,17 @@ func addAppFlags(command *cobra.Command, opts *appOptions) {
 // NewApplicationUnsetCommand returns a new instance of an `argocd app unset` command
 func NewApplicationUnsetCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 	var (
-		parameters  []string
-		valuesFiles []string
+		parameters      []string
+		valuesFiles     []string
+		nameSuffix      string
+		namePrefix      string
+		kustomizeImages []string
 	)
 	var command = &cobra.Command{
-		Use:   "unset APPNAME -p COMPONENT=PARAM",
-		Short: "Unset application parameters",
```

Comment:
Just "Unset application" is not very clear . I think `Unset application parameters` is good enough. We keep referring to config tool settings as "parameters".

Reply:
:+1 Change is checked in.